### DISCUSSION TITLE
Deploy error

### DISCUSSION THREAD

**Discussion by ahmadjonhn** [sandstreamdev/react-swipeable-list#179] [General]
Failed to compile.
--
20:53:40.219 | ModuleNotFoundError: Module not found: Error: Can't resolve '@sandstreamdev/react-swipeable-list/dist/styles.css' in '/vercel/workpath0/pages'

Could you help me to overcome this deployment error?

**Comment by marekrozmus** [comment]
Hi,

Could you please provide some minimal repository with configuration that I could reproduce this issue?

Maybe it is well known problem with vercel: https://stackoverflow.com/questions/62378045/how-to-fix-next-js-vercel-deployment-module-not-found-error ?

**Reply by littlefroginnovations** [reply]
I get the same error. 
I am using expo to run emulator
Says
Componenet Exception
React.cloneElement(...): The argument must be a React Element, but you passed null.

**Reply by marekrozmus** [reply]
Hi,

Could you please provide some minimal repository with configuration that I could reproduce this issue?

**Reply by littlefroginnovations** [reply]
i was using expo and doing this in react-native, i moved on because it seems your package was just for web and not mobile.
